### DISCUSSION TITLE
After activating the workflow, all categories inherit the Basic Workflow

### DISCUSSION THREAD

**Discussion by hytsch** [joomla/joomla-cms#47019] [4. Rejected Features]
### Steps to reproduce the issue
Create multiple categories and subcategories. Activate the workflow, create a new workflow (in addition to the basic workflow), assign this workflow to a main category (instead of the basic workflow).

### Expected result
The subcategories inherit the default and adopt the new workflow.

### Actual result
The subcategories keep the basic workflow as the default.

### System information (as much as possible)
j5.2.2

**Comment by brianteeman** [comment]
I do not share those expectations. If a category has a setting then I would not expect that setting to be changed because you changed a setting somewhere else. 

You do however have the option to make it inherit if you so wish by explicitly setting the subcategory to inherit

![Image](https://github.com/user-attachments/assets/a7e676c6-b8a0-4d7c-994c-cfc43a641c96)

**Comment by hytsch** [comment]
This means that for a larger website we have to manually set the workflow setting to 'inherit' in each subcategory.

**Comment by brianteeman** [comment]
When you create the category then you should set it to inherit if that's what you desire.

**Comment by hytsch** [comment]
Sure, I understand that and I do it that way - on a website where the workflows are already active. I'm talking about a case with an extensive website (>1000 categories) where five separate workflows are to be defined for five main categories with >500 subcategories. And here I had the 'nasty surprise' that all subcategories are set to basic workflow. In my opinion, that's a bad idea because the 'inheritance' use case is the typical one - since the workflow is more likely to be used on extensive websites.

**Comment by rdeutz** [comment]
The Joomla Maintainer Team decided at the 25.02.2026 meeting to reject this feature. A batch job would be a better solution for it.
